File: Projekt/API/API/Models/UserProfile.cs
public class UserProfile
{
    public UserProfile(){}
    public UserProfile(UserProfileDto dto)
    {
        Id = dto.Id;
        UserName = dto.UserName;
        Email = dto.Email;
        Bio = dto.Bio;
    }
    public int Id { get; set; }
    public string UserName { get; set; }
    public string Email { get; set; }
    public string Bio { get; set; }
}

File: Projekt/mobi_app/lib/models/UserProfile.dart
// -- user_profile.dart --
import 'package:json_annotation/json_annotation.dart';

part 'UserProfile.g.dart';

@JsonSerializable()
class UserProfile {
  int id;
  String userName;
  String email;
  String bio;

  UserProfile(this.id,this.userName,this.email,this.bio,);

  factory UserProfile.fromJson(Map<String, dynamic> json) => _$UserProfileFromJson(json);

  Map<String, dynamic> toJson() => _$UserProfileToJson(this);
}